editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser2.tsx

```diff
@@ -91,6 +91,25 @@ function actionsForUpdatedCode(updatedCodeSnippet: string) {
   return [updateAction, requestLintAction]
 }
 
+async function toggleConditional(
+  renderResult: EditorRenderResult,
+  buttonTestId: string,
+  targetPath: ElementPath,
+): Promise<void> {
+  await act(async () => {
```

Comment:
could we use the single undo step checker function here?

Reply:
we can!

Reply:
done in https://github.com/concrete-utopia/utopia/pull/3333/commits/3e8e45faf79cd9480673889952720c7795fa88b8

Reply:
thanks!